### PR TITLE
BE-1: Add server which listens but does nothing

### PR DESCRIPTION
**Pull request by LiskaC**
First additions to have a server.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const server = http.createServer((req, res) => {
+  // TODO - add functionality, this server does not yet do anything.
+})
+
+const PORT = 3000
+const HOST = 'localhost'
+
+server.listen(PORT, HOST, () => {
+  console.log(`Server running at http://${HOST}:${PORT}/`)
```
Comment (markuskappel) [inline server/server.js]: Better log this on log level `info`.

**Review by markuskappel — COMMENTED** [review]
LGTM. Just a little comment on the log level

File: server/server.js (hunk 1 above)
Comment (LiskaC) [reply server/server.js]: thanks m8! will do

**Review by markuskappel — APPROVED** [review]
LGTM!

**Pull request merged**